libdnf/hy-util.cpp

```diff
@@ -118,7 +118,7 @@ hy_detect_arch(char **arch)
             modifier++;
         if (getauxval(AT_HWCAP) & HWCAP_ARM_VFP)
             *modifier++ = 'h';
-        if (getauxval(AT_HWCAP2) & HWCAP2_AES)
+        if ((atoi(un.machine+4) >= 8) && (getauxval(AT_HWCAP2) & HWCAP2_AES))
```

Comment:
Please write a comment about what's going on here.
It's clear from the commit message, but even a single change on the line would change `dnf blame` output and hide it from the user.